### PR TITLE
Release/public-v3: Add RT tests for WoFS_v0 suite for SRW-v3

### PR DESCRIPTION
**Pull request by MicroTed**
# PR Checklist

- [x] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [ ] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Description

Adds RT for wofs_v0 test (regional_3km_wofs) for SRW app v3

### Issue(s) addressed

Cherry-picked previous commits adding tests for WoFS_v0 suite. Note that the decomposition and restart tests are currently known to fail, for reasons outside of the suite physics (when CCPP update for NSSL microphysics is updated). (Decomp test will work if nrows_blend = 0 instead of 10)

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [ ] hera.gnu
- [ ] orion.intel
- [ ] cheyenne.intel 
- [ ] cheyenne.gnu
- [ ] gaea.intel 
- [ ] jet.intel
- [ ] wcoss_cray
- [ ] wcoss_dell_p3
- [ ] opnReqTest for newly added/changed feature
- [ ] CI

## waiting for other repos
- waiting on NOAA-EMC/fv3atm/pull/535
- Should get CCPP update for bug fixes in NSSL scheme (for restarts and decomposition tests)

### CODE REVIEW

**Review by RatkoVasic-NOAA — APPROVED** [review]

**Comment by jkbk2004** [comment]
@MicroTed @RatkoVasic-NOAA @DusanJovic-NOAA @MinsukJi-NOAA SRW @daniel-hildebrandt This PR is for the release/public-v3 SRW v2 release. SRW group agreed to merge the PR w/o RT tests. Extensive SRW tests will go on after SRW release branch gets frozen. Should we move on merging the PR? Review comments will help to expedite the SRW group's request.

**Comment by MinsukJi-NOAA** [comment]
@MicroTed Have the ORT-related changes been tested?

**Comment by DusanJovic-NOAA** [comment]
`.gitmodules` still points to MicroTed's fv3atm fork. Is that what you want?

**Comment by jkbk2004** [comment]
> `.gitmodules` still points to MicroTed's fv3atm fork. Is that what you want?

@DusanJovic-NOAA fv3atm needs to point to release/public-v3 @MicroTed would you correct the pointer?

**Comment by jkbk2004** [comment]
> @MicroTed Have the ORT-related changes been tested?

@MinsukJi-NOAA @MicroTed I am not sure specifically about this PR itself but ORT tests were working ok nblend=0, @MicroTed right? There is a separate PR to follow up this feature on develop branch.

**Comment by MicroTed** [comment]
@jkbk2004 I updated .gitmodules and the FV3 pointer.

Regarding ORT: Yes, decomposition works with nrows_blend=0, and a fix is underway for values of 1-10 (thanks to @junwang-noaa and @RatkoVasic-NOAA). I'm not sure about the status of restart for regional -- that was known to be faulty.

**Comment by MinsukJi-NOAA** [comment]
> 

Thanks @MicroTed. I saw file changes for ORT related scripts, so my comments were more about that than actually passing the ORT's.

File: .gitmodules [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-EMC/fv3atm
-  branch = develop
+  url = https://github.com/NOAA-EMC/fv3atm.git
```
Comment (DusanJovic-NOAA) [inline .gitmodules]: Please remove `.git` to be consistent

Comment (MicroTed) [reply .gitmodules]: Whoops, thanks for noticing that. Fixed

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by MinsukJi-NOAA — APPROVED** [review]

**Pull request merged**